clients/cobol-lsp-vscode-extension/src/__tests__/services/util/ConfigurationWatcher.spec.ts

```diff
@@ -34,6 +34,12 @@ jest.mock("vscode", () => ({
       .fn()
       .mockReturnValue("onDidChangeConfiguration"),
   },
+  DiagnosticSeverity: {
```

Comment:
Definitely keep this in the common vscode mock file. As far as I can tell, this whole block is not needed, as everything is already mocked in the common file.

Reply:
Fixed